### ISSUE TITLE
recipe should require libpng>=1.6.28

### ISSUE THREAD

**Issue by agoodm** [conda-forge/matplotlib-feedstock#103]
After installing `matplotlib` 2.0.0 on a system with libpng 1.6.26, importing `pyplot` fails with

```
>>> import matplotlib.pyplot as plt
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/goodman/opt/envs/mpl/lib/python2.7/site-packages/matplotlib/pyplot.py", line 29, in <module>
    import matplotlib.colorbar
  File "/Users/goodman/opt/envs/mpl/lib/python2.7/site-packages/matplotlib/colorbar.py", line 36, in <module>
    import matplotlib.contour as contour
  File "/Users/goodman/opt/envs/mpl/lib/python2.7/site-packages/matplotlib/contour.py", line 23, in <module>
    import matplotlib.text as text
  File "/Users/goodman/opt/envs/mpl/lib/python2.7/site-packages/matplotlib/text.py", line 33, in <module>
    from matplotlib.backend_bases import RendererBase
  File "/Users/goodman/opt/envs/mpl/lib/python2.7/site-packages/matplotlib/backend_bases.py", line 63, in <module>
    import matplotlib.textpath as textpath
  File "/Users/goodman/opt/envs/mpl/lib/python2.7/site-packages/matplotlib/textpath.py", line 20, in <module>
    from matplotlib.mathtext import MathTextParser
  File "/Users/goodman/opt/envs/mpl/lib/python2.7/site-packages/matplotlib/mathtext.py", line 62, in <module>
    import matplotlib._png as _png
ImportError: dlopen(/Users/goodman/opt/envs/mpl/lib/python2.7/site-packages/matplotlib/_png.so, 2): Library not loaded: @rpath/libpng16.16.dylib
  Referenced from: /Users/goodman/opt/envs/mpl/lib/python2.7/site-packages/matplotlib/_png.so
  Reason: Incompatible library version: _png.so requires version 45.0.0 or later, but libpng16.16.dylib provides version 44.0.0
```

I have tested and confirmed that the actual dependency should be libpng>=1.6.28, not 1.6.23.

**Comment by astrofrog** [comment]
I can confirm that I'm getting this too after upgrading to Matplotlib 2.0

**Comment by agoodm** [comment]
Should be resolved by #104 .
